tools/deployment_manager/configs/darwin.json

```diff
@@ -20,7 +20,7 @@
         "runtime/lib/intel64/libinference_engine_transformations.dylib",
```

Comment:
can we also remove prefixes in file names like `hetero_plugin.hpp`? I suppose all plugin should follow this scheme

Reply:
Do you mean inference_engine prefix?